src/VERSION

```diff
@@ -1 +1 @@
-0.32.0
+0.33.0-dev
```

Comment:
```suggestion
0.32.1-dev
```

Reply:
Since this is based on master, it should be 0.33.0-dev.
The release/0.32 branch will have `0.32.1-dev` until it is released. And will be using 0.31.1 for building the compiler, instead of 0.32.0.